vaas-app/src/vaas/cluster/models.py

```diff
@@ -53,6 +56,9 @@ def __str__(self):
     def get_template_version(self):
         return self.version
 
+    def clean(self):
+        vcl_variable_validator(self.content, self.pk, VclVariable, VarnishServer)
```

Comment:
is this correct place for injecting validators ?

Reply:
According to Django docs: "This method should be used to provide custom model validation" .

Reply:
HINT: a model’s clean() method is not invoked when you call your model’s save() method.